homeassistant/components/nzbget/sensor.py

```diff
@@ -18,6 +18,7 @@
     "download_rate": ["DownloadRate", "Speed", "MB/s"],
     "download_size": ["DownloadedSizeMB", "Size", "MB"],
     "free_disk_space": ["FreeDiskSpaceMB", "Disk Free", "MB"],
+    "post_job_count": ["PostJobCount", "Post Processing Jobs", "Count"],
```

Comment:
Counts are mostly `#`. What do you thing?

Reply:
This value from NZBGet is a number, as you can see in my screenshot above. I took some inspiration from the [Glances component](home-assistant/homeassistant/components/glances/const.py) where `Count` is consistently used for something like the number of running processes.

But I'm open for suggestions if you have any.

Reply:
`Jobs` seems appropriate, that's what you're counting.

Reply:
@balloob Thanks. I've amended this in my recent commit.